INSTALL

```diff
@@ -107,3 +107,15 @@ NOTES
 		the first time you run it because GNU Make fails inference rules
 		when directories are not created yet. Try running make after the
 		failure.
+
+
+HINT FOR BUILD IN DEBIAN/UBUNTU
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For building under Debian 16, targeting ImageMagick 6 (provided patch #73 is applied):
```

Comment:
About the text '(provided patch #73 is applied)' , I would prefer

1) either provide full link, like: https://github.com/techblue/jmagick/issues/73
2) or, even better, simply make a PR and lets merge that PR first